distribution/lib/Standard/Table/0.0.0-dev/src/Simple_Expression.enso

```diff
@@ -210,6 +219,8 @@ type Math_Operation
             Math_Operation.Ceil -> column.ceil
             Math_Operation.Floor -> column.floor
             Math_Operation.Truncate -> column.truncate
+            Math_Operation.Absolute -> (column:Numeric_Column).abs
+            Math_Operation.Signum -> (column:Numeric_Column).signum
```

Comment:
I wonder if we want a more gracious error here than just a conversion failed error?

Reply:
Yes that is a good point.